sdk/python/foundation-models/system/inference/image-instance-segmentation/image-instance-segmentation-batch-endpoint.ipynb

```diff
@@ -127,7 +127,7 @@
    "source": [
     "### 2. Pick a model to deploy\n",
     "\n",
-    "Browse models in the Model Catalog in the AzureML Studio, filtering by the `image-instance-segmentation` task. In this example, we use the `mask_rcnn_swin-t-p4-w7_fpn_1x_coco` model. If you have opened this notebook for a different model, replace the model name and version accordingly. This is a pre-trained model and may not give correct prediction for your dataset. You should probably TRAIN this model on a down-stream task to be able to use it for predictions and inference. Please refer to the [notebook.](../../finetune/image-instance-segmentation/mmdetection-fridgeobjects-instance-segmentation.ipynb)"
+    "Browse models in the Model Catalog in the AzureML Studio, filtering by the `image-instance-segmentation` task. In this example, we use the `mask_rcnn_swin-t-p4-w7_fpn_1x_coco` model. If you have opened this notebook for a different model, replace the model name accordingly. This is a pre-trained model and may not give correct prediction for your dataset. We strongly recommend to TRAIN this model on a down-stream task to be able to use it for predictions and inference. Please refer to the [notebook.](../../finetune/image-instance-segmentation/mmdetection-fridgeobjects-instance-segmentation.ipynb)"
```

Comment:
[Minor] We strongly recommend to TRAIN -> We strongly recommend to finetune

since in our folder structure also, we keep the notebooks under finetune folder.

Reply:
Will address in the following.